cpp/include/Ice/LocalExceptions.h

```diff
@@ -626,7 +628,7 @@ namespace Ice
         [[nodiscard]] const char* ice_id() const noexcept final;
     };
 
-    /// This exception indicates that an attempt has been made to change the connection properties of a fixed proxy.
+    /// The exception that is thrown when attempting to change the connection properties of a fixed proxy.
```

Comment:
is this accurate? for example:

```
ReferencePtr
IceInternal::FixedReference::changeAdapterId(string /*newAdapterId*/) const
{
    throw FixedProxyException(__FILE__, __LINE__);
}
```

That is not a connection property.

Reply:
It is connection-related. How would you describe it?

Reply:
New version:
```
/// The exception that is thrown when attempting to change a connection-related property on a fixed proxy.
```